### ISSUE TITLE
Parallelise deployment of API, Celery, and Celery Beat

### ISSUE THREAD

**Issue by philherbert** [department-of-veterans-affairs/notification-api#65]
In our Github Actions workflow, we first deploy the API, then Celery, then Celery Beat. These three components don't depend on each other, so we can split these into separate jobs and run them in parallel.

Hopefully this will speed up the build a little 🤞

**Comment by philherbert** [comment]
Some uncertainty about rollbacks - if any of the deployments fail, successful deployments would not be rolled back. However, this is the case now (with sequential deployments), so we wouldn't be any worse off for doing this.

We should have a deeper think about strategies for rollbacks.

**Comment by saman-moshafi-va** [comment]
The build became 4-6 minutes faster.
